.github/workflows/firebaseai.yml

```
@@ -0,0 +1,48 @@
+name: vertexai
+
+on:
+  pull_request:
+    paths:
+    - 'firebaseai/**'
+  schedule:
+    # Run every day at 11pm (PST) - cron uses UTC times
+    - cron:  '0 7 * * *'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+env:
+  SAMPLE: vertexai
```

I'm not certain where `SAMPLE` is used but looks like all the workflows have it. If it still builds, should it be `firebaseai`?
```suggestion
  SAMPLE: firebaseai
```